en/subscription_services_privacy_notice.md

```diff
@@ -1,55 +1,67 @@
 
-This privacy notice explains what data Firefox Relay and Mozilla VPN collect, share, and why. We also adhere to the [Mozilla Privacy Policy](https://www.mozilla.org/privacy/) for how we receive, handle, and share information.
+This privacy notice explains what data Mozilla VPN, Mozilla Monitor and Firefox Relay collect, share, and why. We also adhere to the [Mozilla Privacy Policy](https://www.mozilla.org/privacy/) for how we receive, handle, and share information.
 
 ## Things You Should Know
 
 __Mozilla account information.__ These Services require a Mozilla account, which sends Mozilla your email address, locale, and IP address. Learn more about [Mozilla Account data practices](https://www.mozilla.org/privacy/mozilla-accounts/).
 
-__Payment information.__ When you subscribe to the Services, you will send payment through one of our third-party payment providers: Stripe, Apple, PayPal, or Google Pay. Mozilla receives a record of your account (including your billing address and the last four digits of your payment method) and the status of your account’s subscription. Mozilla does not store your full payment details.
+__Payment information.__ When you subscribe to Services that require a paid subscription, you will send payment through one of our third-party payment providers: Stripe, Apple, PayPal, or Google Pay. Mozilla receives a record of your account (including your billing address and the last four digits of your payment method) and the status of your account’s subscription. Mozilla does not store your full payment details.
 
 ### Mozilla VPN {: #vpn }
 
-__Interaction data.__ Mozilla receives data about your interactions with Mozilla VPN. For example, we would be able to know things like if you’ve excluded any Apps from protection (but not which ones), in how many devices you are using your Mozilla VPN Subscription, and whether or not you’ve enabled the ads blocking feature. For a full review of our telemetry, you can learn more in [VPN interaction data](https://dictionary.telemetry.mozilla.org/apps/mozilla_vpn). We use Interaction data to improve performance and stability for our users, and to measure the Services’ performance.
+__Interaction data.__ Interaction data collection is disabled by default. If you opt-in, Mozilla will receive data about your interactions with Mozilla VPN. For example, we would be able to know things like if you’ve excluded any Apps from protection (but not which ones), in how many devices you are using your Mozilla VPN Subscription, and whether or not you’ve enabled the ads blocking feature. For a full review of our telemetry, you can learn more in [VPN interaction data](https://dictionary.telemetry.mozilla.org/apps/mozilla_vpn). We use Interaction data to improve performance and stability for our users, and to measure the Services’ performance.
```

Comment:
- `opt-in` -> `opt in`
- Lower case `i` as in `Interaction data`?
- `Services’` or lower case `s` when used alone?